tests/fixtures/fixtures_order.py

```diff
@@ -19,6 +183,21 @@ def order_mock(auth_mock, requests_mock):
     ) as json_file:
         json_oder_schenma = json.load(json_file)
         requests_mock.get(url=url_order_info, json=json_oder_schenma)
+
+    requests_mock.get(url=URL_STAC_CATALOG, json=JSON_STAC_CATALOG_RESPONSE)
+
+    url_asset_stac_info = f"{auth_mock._endpoint()}/v2/assets/stac/search"
+    requests_mock.post(
+        url_asset_stac_info,
+        [
+            {"json": JSON_ORDER_STAC},
```

Comment:
This depends on the orders of calls and is no-go for shared fixture, let's break this to 3 different order mocks